kitsune/questions/static/questions/js/constants/AAQConstants.es6.js

```diff
@@ -1,28 +0,0 @@
-import constantMap from '../../../sumo/js/utils/constantMap.es6.js';
```

Comment:
constantMap is only imported in UrlConstants which is only imported in UrlActions and UrlStore, they can all be removed